sig-docs/annual-activities.md

```diff
@@ -0,0 +1,84 @@
+# List of annual activities to be performed by SIG Docs leadership
+
+This document lists and details the activities to be performed annually by [SIG Docs leadership](./sig-docs/README.md#leadership)
+
+The recommended timeline for these activities is usually November - January due to lower levels of activity.
+
+## Prune OWNERS_ALIASES under k/website
+
+At the end of each year, leadership needs to ensure that folks listed in the [OWNERS_ALIASES file](https://github.com/kubernetes/website/blob/main/OWNERS_ALIASES) 
+are willing to continue contributing.
+
+This can be done by using the [maintainers tool](https://github.com/kubernetes-sigs/maintainers) which helps parse the file for 
+activity levels of those listed from GitHub and [devstats](https://k8s.devstats.cncf.io).
+
+### Steps
+
+- Set up the [maintainers tool](https://github.com/kubernetes-sigs/maintainers) along with its prerequisites locally by following the instructions.
+- Clone the [k/website](https://github.com/kubernetes/website) repository on your machine.
+- Create a new branch locally. This is required since there are nested OWNERS_ALIASES files within the repository.
+- Switch to the newly created branch and delete all nested OWNERS_ALIASES files. Retain the OWNERS_ALIASES file at the top of the repository root only.
+Without this, the maintainers tool will not produce the required output.
+- Run the maintainers tool against this new copy of the website repo locally.
+  - Tip: While executing the command, ensure you exclude the following folks by using the `--exclude` flag:
+      - Security Response Committee members,
+      - Release engineering approvers and reviewers,
+      - Co-chairs and tech leads of SIG Docs, SIG Release, and SIG Security
+- You'll get a list of SIG Docs approvers/reviewers with no contributions and with low reviews/approvals.
+- Submit a separate PR each for the list with no contributions and with low reviews/approvals and seek feedback. The timeframe for this activity should be ~2 weeks per PR.
+  - Tip: The lists will likely be long. It'd be easier if you raise the second one after the first PR is merged.
+  - Example PRs:
+    - https://github.com/kubernetes/website/pull/38719
+    - https://github.com/kubernetes/website/pull/38853
+- Socialize the activity within the [SIG Docs google group](https://groups.google.com/u/1/g/kubernetes-sig-docs) and the [#sig-docs slack channel](https://kubernetes.slack.com/messages/sig-docs). Add it as an item to the upcoming meeting agenda.
+- Work with all stakeholders to get the PRs merged, ideally before the end of the year.
+- In case of issues with merging due to conflicts with localization initiatives, such as lack of minimum number of approvers/reviewers,
+seek consensus with the localization subproject and wider SIG Docs community in the next year.
+  
+## Create new PR Wrangler schedule
```

Comment:
I think it might be helpful to mention the weeks and holidays to leave unscheduled here

Reply:
Thanks, this is a good point! I'll address this in a follow-up PR.